langchain/chains/llm.py

```diff
@@ -30,6 +36,7 @@ class LLMChain(Chain, BaseModel):
     """Prompt object to use."""
     llm: BaseLanguageModel
     output_key: str = "text"  #: :meta private:
+    guarded_output_parser: Optional[GuardedOutputParser] = None
```

Comment:
i know this diff is meant to be minimal (and hence not including that discussion around whether prompt should have the parser). but now in examples we show, users will have to set a parser twice basically. do you think its reasonable to take it out of the prompt and set it here. and set only one of an OutputParser or a GuardedOutputParser?